addon/register-units.js

```diff
@@ -93,3 +94,15 @@ registerUnit(['psi'], psi);
 
 let atmosphere = Unit.scale(101325, pascals);
 registerUnit(['atmosphere', 'at'], atmosphere);
+
+// Volume
+
+let liters = Unit.root();
+registerUnit(['litres', 'liters', 'L', 'l'], liters);
+registerUnit(['millilitres', 'milliliters', 'ml', 'mL'], Unit.scale(1e-3, liters));
+registerUnit(['gallons', 'gal'], Unit.scale(3.78541, liters));
+registerUnit(['quart', 'qt'], Unit.scale(0.946353, liters));
+registerUnit(['pint', 'pt'], Unit.scale(0.473176, liters));
+registerUnit(['fluid ounces', 'floz'], Unit.scale(0.0295735, liters));
```

Comment:
In order to improve accuracy among similar units, you should fork units that have simple conversions off of eachother, e.g.

```js
let fluidOunces = Unit.scale(0.0295735, liters);
registerUnit(['fluid ounces', 'fl oz'], fluidOunces);
registerUnit(['pint', 'pt'], Unit.scale(16, fluidOunces));
registerUnit(['quart', 'qt'], Unit.scale(32, fluidOunces));
registerUnit(['gallons', 'gal'], Unit.scale(128, fluidOunces));
```

It also makes it easier to verify :)